ui-asset-management/api/svc-asset-details/extension/clients/asset_details_client.go

```
@@ -138,13 +144,18 @@ func (c *AssetDetailsClient) extractTags(assetDetails map[string]interface{}) (m
 	return tags, nil
 }
 
+func getTagDisplayName(tag string, mandatoryTags []models.Tag) string {
+	for _, mandatoryTag := range mandatoryTags {
+		if strings.ToLower(tag) == strings.ToLower(mandatoryTag.TagName) {
```

golangci-lint

SA6005: should use strings.EqualFold instead

(staticcheck)